main.py

```diff
@@ -1 +1,207 @@
+    for _ in range(3):
+        while True:
+            coordi_weed_x = random.randint(0, row - 1)
+            coordi_weed_y = random.randint(0, col - 1)
+            if garden[coordi_weed_x][coordi_weed_y] == EMPTY_CELL:
+                garden[coordi_weed_x][coordi_weed_y] = WEED
+                break
+
+    # Loop for 1/10 chance of every square being either a flower or Weed
+    for r in range(row):
+        for c in range(col):
+            roulette = random.randint(1, 10)
+            if roulette == 1:
+                random_fifty = random.randint(1, 2)
+                if random_fifty == 1:
+                    garden[r][c] = FLOWER
+                elif random_fifty == 2:
+                    garden[r][c] = WEED
+
+    return garden
+
+
+# function to display the garden
+
+
+def display_garden(garden, horizontal, vertical):
+    """Return a grid of n by m squares"""
+    for r in range(horizontal):
+        for c in range(vertical):
+            print(garden[r][c], end="")
+        print("")
+
+def plant_spread(garden , spread_list , item):
+    # try:
+    #     if garden[coordi_spread_x ][coordi_spread_x] == EMPTY_CELL:
+    #        garden[coordi_spread_x][coordi_spread_y] = item
+    # except IndexError :
+    #     pass 
+    # return garden
+
+    for x, y in spread_list:
+        if 0 <= x < len(garden) and 0 <= y < len(garden[0]):
+            if garden[x][y] == EMPTY_CELL:
+                garden[x][y] = item
+
+
+
+# counts the total number of flowers and weeds
+def counting_items(garden, horizontal, vertical):
+    """Return the number of flowers and weeds in the garden"""
+    flower_count = 0
+    weed_count = 0
+    for rows in range(horizontal):
+        for column in range(vertical):
+            if garden[rows][column] == FLOWER:
+                flower_count += 1
+            elif garden[rows][column] == WEED:
+                weed_count += 1
+    return flower_count, weed_count
+
+
+# Ensures fulfillment of condittions with eachh passing day in the simulation where each flower and weed takes over adjacent squares
+
+
+def day_increment(garden, horizontal, vertical):
+    """Returns a version of the garden where a day has passed"""
+    # for row in range(horizontal):
+    #     for col in range(vertical):
+    #         if garden[row][col] == FLOWER:
+    #             item = FLOWER
+    #             garden = plant_spread(garden, row -1, col, item)
+    #             garden = plant_spread(garden, row +1, col, item)
+    #             garden = plant_spread(garden, row, col -1, item)
+    #             garden = plant_spread(garden, row, col + 1, item)
+    #         elif garden[row][col] == WEED:
+    #             item = WEED
+    #             garden = plant_spread(garden, row - 1, col, item)
+    #             garden = plant_spread(garden, row + 1, col, item)
+    #             garden = plant_spread(garden, row, col - 1, item)
+    #             garden = plant_spread(garden, row, col + 1, item)
```

Comment:
Never leave comments like these on the remote repo. Version control is used for exactly this purpose that we can revert to last stable release anytime without needing to leave commented code.